packages/fuels-abigen-macro/tests/test_projects/string_and_array_inside_enum/src/main.sw

```diff
@@ -0,0 +1,21 @@
+contract;
```

Comment:
refactor to include inside the `xx_inside_enum` so we have one "big project" that has all the corner cases instead of several small ones. We are starting to have to many small test projects.

Reply:
In that particular case it just makes much more sense to have every `xxx_inside_enum` inside one project, that would be the "goto" place for that kind of issue. It makes the directory structure less hell to navigate. But you're right that it shouldn't be an optimisation criteria, but here I'm deeply conviced it makes a lot of sense.